mrp_operations_extension/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+MRP operations extension module
+===============================
+
+This module adds:
+
+* New table to store operations to avoid typing them again.
+* Adds a relation from workcenter lines to BoM lists.
+* Adds a relation from workcenter lines to Manufacturing Orders in
```

Comment:
"A relation between workcenter lines and Manufacturing Orders in
  Scheduled/Consumed/Finished products."